nosso-calendario/src/main/java/com/br/devefc/nossocalendario/model/User.java

```diff
@@ -0,0 +1,55 @@
+package com.br.devefc.nossocalendario.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+	@Id
+	@GeneratedValue(strategy=GenerationType.AUTO)
+	private Long id;
+
+	private String login;
+	private String senha;
+	private LocalDateTime acesso_cadastro;
+	
+	public User() {
+	}
+	
+	public User(String login, String senha, LocalDateTime acesso_cadastro) {
+		this.login = login;
+		this.senha = senha;
+		this.acesso_cadastro = acesso_cadastro;
+	}
+
+	public Long getId() {

```

Comment:
*A complicação do nosso é código é proporcional a complicação da nossa feature. Quanto mais simples, melhor.*

Se tem método público na sua classe é porque alguém, provavelmente vai chamar. O que não é o caso neste momento do seu código. Minha sugestão aqui é apagar e esperar a necessidade aparecer :).

Reply:
Optei por deixar disponível somente o método **getId()**, mais por causa da visualização no swagger e caso for escrever algum teste eu validar o retorno se foi salvo ou não por meio do id retornado

> _A complicação do nosso é código é proporcional a complicação da nossa feature. Quanto mais simples, melhor._
> 
> Se tem método público na sua classe é porque alguém, provavelmente vai chamar. O que não é o caso neste momento do seu código. Minha sugestão aqui é apagar e esperar a necessidade aparecer :).

Optei por deixar disponível somente o método **getId()**, mais por causa da visualização no swagger e caso for escrever algum teste eu validar o retorno se foi salvo ou não por meio do id retornado